css/main.css

```diff
@@ -27,3 +27,14 @@ mark {
 .indented-level3 {
     padding-left: 60px;
 }
+
+/* Responsive table of contents (toc) */
+
+/* Hides toc on screen widths smaller than 1300px */
+/* page-nav appears on screens 1300px wide and above */
+@media screen and (min-width: 1299.98px) {
```

Comment:
Did you have any particular reason for not using 1300px? I'm thinking it might work more smoothly with page-nav because now there's a little sweet spot where neither are shown:

<img width="1411" alt="Screenshot 2020-02-21 at 9 47 46 AM" src="https://user-images.githubusercontent.com/42177597/74997009-0ee2f480-5490-11ea-9a70-734cb9952cde.png">

Reply:
I wanted to be consistent with `page-nav.css` which has the following:
```
/* Responsive page navigation */

/* Hides page navigation on screen widths smaller than 1300px */
@media screen and (max-width: 1299.98px) {
    #page-nav {
        border: none;
        overflow: hidden;
        padding: 0;
        width: 0;
    }
}
```

But I'm curious, how did you manage to get both the toc and page-nav to disappear?

Reply:
> I wanted to be consistent with `page-nav.css` which has the following:

Ah okay that makes sense, thanks for the clarification.

> But I'm curious, how did you manage to get both the toc and page-nav to disappear?

I just messed around with screen widths haha, but honestly it's an edge case and not expected user behaviour so my personal opinion is that it should be fine

![Feb-21-2020 15-01-55](https://user-images.githubusercontent.com/42177597/75011743-5763d780-54bb-11ea-9c03-435295557520.gif)

Reply:
> I just messed around with screen widths haha, but honestly it's an edge case and not expected user behaviour so my personal opinion is that it should be fine

Ah I see. I manually inputted 1299 and 1300 into the dev tools to test since it only accepts integer values.

I agree that it is just an edge case, most users won't encounter this